content/applications/finance/accounting/payments.rst

```diff
@@ -30,114 +32,149 @@ At this point, the customer invoice or vendor bill is marked as :guilabel:`In pa
 the outstanding account is reconciled with a bank statement line, the invoice or vendor bill changes
 to the :guilabel:`Paid` status.
 
-The information icon near the payment line displays more information about the payment. You can
-access additional information, such as the related journal, by clicking :guilabel:`View`.
+The :icon:`fa-info-circle` information icon next to the payment line displays more
+information about the payment. To access additional information, such as the related journal, click
+:guilabel:`View`.
 
 .. image:: payments/information-icon.png
```

Comment:
It would be nice to update this. It can either be done from 17 onward or it can updated now and the 16 documentation will just have a screenshot from a future version, which isn't a big deal for me.

Reply:
@dade-odoo I'll do it in 17 ;)